src/Game/ObjectFactory.cpp

```diff
@@ -135,8 +135,9 @@ Object* ObjectFactory::createObject(unsigned int PID)
             auto msg = ResourceManager::getInstance()->msgFileType("text/english/game/pro_item.msg");
             try
             {
-                object->setName(msg->message(proto->messageId())->text());
-                object->setDescription(msg->message(proto->messageId() + 1)->text());
+				auto o = msg->message(proto->messageId());
```

Comment:
I am not using tabs, I am using enter, VS2017 does it whenever I reinsert removed lines.